### PR TITLE
Update demo app to use adwaita theme 0.0.4

### PR DESCRIPTION
**Pull request by pablojimpas**

### CODE REVIEW

File: example/lib/flap/flap_demo.dart [hunk 1]
```diff
@@ -14,8 +14,8 @@ class FlapDemo extends StatelessWidget {
       valueListenable: themeNotifier,
       builder: (_, ThemeMode currentMode, __) {
         return MaterialApp(
-          theme: adwaita.lightTheme,
-          darkTheme: adwaita.darkTheme,
+          theme: adwaita.AdwaitaThemeData.light(),
```
Comment (MalcolmMielle) [inline example/lib/flap/flap_demo.dart:17]: With adwaita 0.0.4, there is no need for the namespace `adwaita`. Could you remove it :)?

**Review by MalcolmMielle — CHANGES_REQUESTED** [review]
Thanks for upgrading adwaita in the example :D!

I have a tiny tiny remark but otherwise, it's all good for me :)

PS: we're in the process of fixing the linter in #44 so don't worry about that.

**Comment by pablojimpas** [comment]
> Thanks for upgrading adwaita in the example :D!
> 
> I have a tiny tiny remark but otherwise, it's all good for me :)
> 
> PS: we're in the process of fixing the linter in #44 so don't worry about that.

Perfect, I've just cleaned up that so I guess it's ready to be merged.

I ran `flutter analyze` in my local machine both on the root directory and the example directory and everything it's ok so I don't know what happened with GitHub Actions.

Keep up the great work, hopefully I can contribute more meaningful changes in the future!

**Comment by MalcolmMielle** [comment]
Thanks for the PR and addressing the comments :).

I broke the GitHub actions, I'm new to it, hahahaa. Not your fault, hopefully it works now :D

**Review by MalcolmMielle — APPROVED** [review]

**Review by MalcolmMielle — CHANGES_REQUESTED** [review]
Just a comment about the drawer removal :).

LGTM otherwise.

Edit: this comment is for the wrong PR :S. Something happened in my VSCode haha

**Pull request merged**